ColonialMarinesALPHA.dme

```diff
@@ -15,11 +15,12 @@
 #include "code\stylesheet.dm"
 #include "code\world.dm"
 #include "code\__DEFINES\__game.dm"
-#include "code\__DEFINES\_math.dm"

```

Comment:
Wha?

Reply:
It's an old file, when I defined maths.dm, I cut out the many mathematical operations here into the new one, only leaving those bits that are actually off topic.